### ISSUE TITLE
Incorrect case for Style/SafeNavigation

### ISSUE THREAD

**Issue by AlexWayfer** [rubocop-hq/rubocop#6915]
Hello.

There is wrong behavior of `Style/SafeNavigation` cop.

--------

## Expected behavior

No offenses.

## Actual behavior

Offense from `Style/SafeNavigation`.

## Steps to reproduce the problem

```ruby
@providers.find do |service|
  break service.lookup(lat, lng) if service
rescue ZoneNotFoundError, Net::OpenTimeout
  next
end
```

## RuboCop version

```
$ [bundle exec] rubocop -V
0.67.2 (using Parser 2.6.2.1, running on ruby 2.5.5 x86_64-linux)
```

**Comment by rrosenblum** [comment]
Thanks for submitting the issue. I have confirmed that this is a bug. The same issue would occur with `return` and `next` as well. It should be a straight forward fix, but I haven't dug through that code in a while.
